src/Model/Resolver/AttributesWithValue.php

```diff
@@ -34,17 +34,25 @@ class AttributesWithValue implements ResolverInterface
      */
     protected $productRepository;
 
+    /**
+     * @var StoreManagerInterface
+     */
+    private $storeManager;
```

Comment:
FYI please prefer protected fields, so if your class is extended the property will be accessible from ancestor.